src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1122,8 +1120,17 @@ void *os::dll_load(const char *filename, char *ebuf, int ebuflen) {
   event.set_name(filename);
 #endif
 
-  // RTLD_LAZY is currently not implemented. The dl is loaded immediately with all its dependants.
-  void * result= ::dlopen(filename, RTLD_LAZY);
+  // RTLD_LAZY has currently the same behavior as RTLD_NOW
+  // The dl is loaded immediately with all its dependants.
+  int dflags = RTLD_LAZY;
```

Comment:
Looks ok. Maybe it would be safer to use `RTLD_NOW` in case the implementation changes at some point of time.

Reply:
Ok.